### ISSUE TITLE
New git flow

### ISSUE THREAD

**Issue by alyssadelaine** [developmentseed/scoreboard#295]
We now have a develop branch, which we will use to collect features before they are merged into master (and thus pulled onto the production server).

Before changes can be merged into master, they must be accompanied by an updated [CHANGELOG.md](https://github.com/developmentseed/scoreboard/blob/develop/CHANGELOG.md). 

The Changelog will describe the changes and collect them under a version corresponding to the types of updates that have been made. Versioning will follow MAJOR.MINOR.PATCH.

Once the PR is merged to master, two tasks must immediately follow:
- [ ] a git tag corresponding to the version in the Changelog should be applied to master
- [ ] the new master branch will be pulled onto production

A PR should not be merged into master unless the merger is prepared to execute the two above tasks.

cc @kamicut @LanesGood

**Comment by kamicut** [comment]
@alyssadelaine I think the content of this issue could be added to a CONTRIBUTING.md ? Similar to https://gist.github.com/PurpleBooth/b24679402957c63ec426

**Comment by alyssadelaine** [comment]
@kamicut That sounds good to me; let's put that on our list of things to do once we resolve our recent 🐛s
